_studio/hevc_fei/h265_fei/include/mfx_h265_fei_encode_plugin_hw.h

```diff
@@ -49,7 +49,7 @@ namespace MfxHwH265FeiEncode
 
             try
             {
-                tmp_pplg = new H265FeiEncodePlugin(false);
+                tmp_pplg = GetInstance();
```

Comment:
no operator new, no bad_alloc exception, so remove that handle

Reply:
Agree